src/main/scripts/org/reaktivity/specification/sse/id/invalid.query.length/request.rpt

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2016-2020 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect await ROUTED_SERVER
+        "nukleus://streams/sse#0"
+  option nukleus:window 8192
+  option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:beginEx()
+                              .typeId(nukleus:id("http"))
+                              .header(":scheme", "http")
+                              .header(":method", "GET")
+                              .header(":path", "/events?lastEventId=SNKUBUk2eDQ5uyAQNzdq4esgg82sl4vNaTePD9EhNgLMFPlx1nkH0B0eEx6J9GpBe5Tfa87ZQR8ERyjk2YqVIJ7hiOWufjaZR2Bu4lX427Rb8mujdGDgCEEBdtjzMcI4O7Vr0qKapsXNx9EjGPiiZkJIBQYcxoyl502sMARkLMiTVP1mj5XG9tF9k4kwKBO3NMiZUf9414i6cUj4GGdyPO6bOv6HqeYi2LXkhMJK8ZHPPufuxWXmxMML1EFNW2Uigohd")
```

Comment:
Suggest changing rejected `lastEventId` parameter value to `abcdefghijklmnopqurstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789abcdefghijklmnopqurstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789abcdefghijklmnopqurstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789abcdefghijklmnopqurstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789abcdefgh`